.gitignore

```diff
@@ -1 +1,4 @@
 rsconnect/*
+*20230724_MC1_*
```

Comment:
these additional files shouldn't be included in `.gitignore`. seems to be user-specific stuff

Reply:
This is indeed user-specific stuff, will adjust .gitignore accordingly